core/Plugin/ControllerAdmin.php

```diff
@@ -88,13 +88,13 @@ private static function notifyAnyInvalidLicense()
 
     private static function notifyAnyInvalidPlugin()
     {
-        $missingPlugins = \Piwik\Plugin\Manager::getInstance()->getMissingPlugins();
-
-        if (empty($missingPlugins)) {
+        if (!Piwik::hasUserSuperUserAccess()) {
```

Comment:
This is only a boolean access so very fast instead of iterating over all plugins and checking each whether they are loaded or not etc. So for super users this will be faster